packages/report-server/src/routes/helpers.ts

```diff
@@ -39,7 +39,6 @@ export const getRequestedOrgUnitObjects = async (
     throw new Error(`No entities found with codes ${orgUnitCodesInArray}`);
   }
 
-  // If entity is a project
```

Comment:
This seems like a weird change was it from the conflict?

Reply:
I delete it for a purpose since the code can explain itself.

Reply:
Good point :)